include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -1484,14 +1484,14 @@ __parallel_set_op(oneapi::dpl::__internal::__device_backend_tag __backend_tag, _
 //------------------------------------------------------------------------
 
 // Tag for __parallel_find_or to find the first element that satisfies predicate
-template <typename _RangeType>
+template <typename... _Ranges>
 struct __parallel_find_forward_tag
```

Comment:
Does this change have any practical usage in this PR? I was able to find this function being called only with one range.